src/app/FavPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import StyledHeader from '../common/StyledHeader'
+import PlantList from '../plantPage/PlantList'
+import GlobalStyles from '../misc/GlobalStyles'
+import GridList from '../common/GridList'
+import StyledNavButton from '../common/StyledNavButton'
+import BackIcon from '../img/BackArrow.png'
+
+const StyledIcon = styled.img`
+  width: 50px;
+  padding: 10px;
+`
+const StyledNavBackButton = styled(StyledNavButton)`
+  bottom: 4%;
+  left: 9%;
+`
+export default function FavPage({
+  onBookmark,
+  plants,
+  getOptionLabel,
+  selection,
+}) {
+  return (
+    <GridList>
+      <GlobalStyles />
```

Comment:
Isn't the GlobalStyles always imported on the highest Component (in this case App.js)? I'm not sure about it.